scripts/nodes/n0_test

```diff
@@ -30,7 +30,7 @@ rm /tmp/main.*.log
   --protocol-port=18000 \
   --txindex \
   --num-mining-threads=1 \
-  --miner-public-keys=BC1YLg7Bk5sq9iNY17bAwoAYiChLYpmWEi6nY6q5gnA1UQV6xixHjfV \
+  --miner-public-keys=tBCKW665XZnvVZcCfcEmyeecSZGKAdaxwV2SH9UFab6PpSRikg4EJ2 \
```

Comment:
danke. maybe add a comment that says that's the public key associated with the starter-bitclout-seed below?